content/20.treatments.md

```diff
@@ -188,13 +188,121 @@ Summarize the state of the antiviral approach.
+Conversely, zinc deficiency is associated with increased susceptibility to infection in the elderly, which can be prevented when zinc supplements are administered [@doi:10.1093/ajcn/85.3.837].
+Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
+In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a flourescently tagged SARS-CoV) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
+
+###### Summary
+
+While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus further research is imperative.
+However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelyhood of an infection. 
+
+##### Vitamin C
+
+Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 [@doi:10.1002/jmv.25707; @doi:10.20944/preprints202003.0347.v1].
+In particular vitamin C has been considered as a potential prophylactic and therapeutic agent of COVID-19. 
+Vitamin C can also be obtained via dietary sources such as fruit and vegetable or via supplementation.  
+
+###### Potential Mechanisms
+Vitamin C plays a signifiant role in promoting immune function due to its effects on various immune cells. 
+Vitamin C affects inflammation by modulating cytokine production, decreasing histamine levels, enhancing the differentiation and proliferation of T- and B-lymphocytes, increasing antibody levels, and protecting against the negative effects of reactive oxygen species amongst other effects [@doi:10.1155/2014/426740; @doi:10.1007/s00210-013-0880-1; @doi:10.3390/nu9111211]. 
+During viral infections vitamin C is utilised as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C. Post infection these levels return to baseline ranges [@doi:10.1177/003693307301800102; @doi:10.1111/j.1749-6632.1975.tb29312.x; @doi:10.1016/0006-2944(79)90058-9; @doi:10.3390/nu9040339; @doi:10.1079/bjn19920004]. 
+
+###### Current Evidence
+
+Despite the fact that it is well documented that regular vitamin C supplementation can reduce the duration of the common cold, vitamin C (> 200 mg) fails to reduce its incidence [@doi:10.1002/14651858.CD000980.pub4].
+Vitamin C has been shown to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia [@doi:10.1097/00006454-199709000-00003].
```

Comment:
```suggestion
Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia [@doi:10.1097/00006454-199709000-00003].
```
My outsider's understanding is that plenty of studies have reported effects of Vitamin C but many don't hold up at the meta-analysis level -- if that's correct, I think it would be good to distinguish meta-analyses from studies, as you do in the lines below